en/SUMMARY.md

```diff
@@ -11,6 +11,7 @@
   * [GitHub](installation/README.md#github-account)
   * [PythonAnywhere](installation/README.md#pythonanywhere-account)
 * [Installation (chromebook)](chromebook_setup/README.md)
+* [Installation (RunCode Cloud)](cloud_development_setup/README.md)
```

Comment:
I think in terms of organisation we should maybe have one page for Installation - use this to explain that there are two ways to do this (explain the fiferences and pros/cons of each) then from there have the two installation guides as sub-pages so people pick one or the other, before miving to the next step. It will hopefully stop people getting confused and trying to do both installation setup guides.